fiftyone/utils/sam3.py

```
@@ -0,0 +1,650 @@
+                    label=label,
+                    bounding_box=bounding_box,
+                    mask=mask_crop,
+                    confidence=float(score),
+                    index=int(obj_id),
+                )
+            )
+
+        return detections
+
+
+def _get_image_size(img: Any) -> Tuple[int, int]:
```

Ruff (0.15.2)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `img`

(ANN401)